client/src/components/AddProposalTeacher.jsx

```diff
@@ -170,10 +170,6 @@ function AddProposalTeacher(props)
     if(invioForm==true)
```

Comment:
In this file I think that we can take the green part because I think that are the updates which you did yesterday

Reply:
Maybe oscar did
I didn't touch client side, but could be bc he added something